src/styles/sidebar.css

```diff
@@ -0,0 +1,39 @@
+.sidebar {
+    background-color: #364150;
+    text-transform: uppercase;
+    min-width: 250px;
+    max-width: 250px;
+    min-height: 100vh;
+}
+.sidebar__list {
+    list-style: none;
+    display: block;
+}
+
+.sidebar__list hr {
+    color: darkgray;
+    width: 150px;
+    margin: 0 0 20px 0; 
+}
+
+.sidebar__list .item {
+    margin-bottom: 20px;
+}
+
+
+.sidebar__list .item a {
+    text-decoration: none;
+    color: rgb(167, 162, 162);
+    font-size: 24px;
+    
+}
+
+.sidebar__list .item a:hover {
+    color: rgb(219, 212, 212);
+    
```

Comment:
There is a lot of codestyle issues in css.
Empty lines, inconsistent spaces between declarations.

It gives bad first impression.
Please fix that, and add tralling empty line in the end of the files where it's missing

Reply:
I rewrote everything to sass, and improved in next branches.

Reply:
I rewrote everything to sass, and improved in next branches.